doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,19 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		func _ready():
+		  var spin_box = SpinBox.new()
+		  get_tree().get_root().add_child(spin_box)
```

Comment:
There's no reason to add it as child of root, it should simply be `add_child(spin_box)` (so it's a child of the current scene).